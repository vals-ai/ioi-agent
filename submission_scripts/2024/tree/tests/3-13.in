version https://git-lfs.github.com/spec/v1
oid sha256:6396a1d4ca754936a268fbaaeeb757c28a40c8f3a2ae95cdc74702f49a5e1133
size 760820
