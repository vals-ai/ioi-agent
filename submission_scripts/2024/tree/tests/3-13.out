version https://git-lfs.github.com/spec/v1
oid sha256:a6d027710a511704dbff068e9a467a4e01e898044794a5494a433a96ad1bf825
size 207
